doc.go

```diff
@@ -455,6 +455,41 @@ of the returned value:
 	    }
 	}
 
+## Arrow batches
+
+There is a possibility to retrieve data in a columnar format similar to native server response.
+You have to use a proper `WithArrowBatches` context, example:
+
+	    var rows driver.Rows
+		err = conn.Raw(func(x interface{}) error {
+			rows, err = x.(driver.QueryerContext).QueryContext(ctx, query, nil)
+			return err
+		})
+
+		...
+
+		batches, err := rows.(sf.SnowflakeRows).GetArrowBatches()
+
+		... // use arrow records
+
+Limitations:
+
+1. When a query regards something that is not real data, it is returned usually in JSON format. It can't be used with arrow batches context. Examples: `SHOW PARAMETERS`.
+2. Snowflake handles timestamps in a range which is higher than available space in Arrow timestamp type. Because of that special treatment should be used (see below).
+
+### Handling timestamps in Arrow batches
+
+Snowflake returns timestamps natively (from backend to driver) in multiple formats.
+Arrow timestamp type is only 8 bytes long and is not enough to cover the whole date/time Range Snowflake handles.
+Also, Snowflake allows for precision of every fraction of seconds up to 9 (nano), whereas Arrow timestamp allows only 3 (milli), 6 (micro) and 9 (nano) precision.
+Because of that, Snowflake uses custom timestamp format in Arrow which differs on timestamp type and precision.
+
+If you want to use timestamps in arrow batches, you have two options:
+
+ 1. Go driver can reduce timestamp struct into simple Arrow Timestamp, if all timestamp values fit into Arrow timestamp field.
+ 2. You can use native Snowflake values. In that case you will receive complex structs as described above. To transform Snowflake values into Golang time.Time you can use `ArrowSnowflakeTimestampToTime`.
```

Comment:
In that case, ... To transform Snowflake values into the Golang `time.Time` struct, you can use the `Arrow..` function.